config/site_settings.yml

```diff
@@ -2970,6 +2970,12 @@ tags:
   min_trust_to_create_tag:
     default: "3"
     enum: "TrustLevelAndStaffSetting"
+  create_tag_allowed_groups:
+    default: "1|3|13"
```

Comment:
Admins, staff, TL3.

Reply:
I don't _think_ this is necessary because admin and staff will already be in the TL3 auto group; however I also don't think it's a bad thing to do either since the old setting was `TrustLevelAndStaffSetting` :+1:

Reply:
I suspect you are correct. The reason I did this is so we can lean maximally on existing tests, which will fail without this. Making too many changes to both tests and code at once always feels a bit shady. 😅